Makefile.am

```diff
@@ -108,7 +108,25 @@ libuv_la_SOURCES += src/unix/async.c \
 endif  # WINNT
 
 EXTRA_DIST = test/fixtures/empty_file \
-             test/fixtures/load_error.node
+             test/fixtures/load_error.node \
+             src/unix/uv-dtrace.d \
+             include \
```

Comment:
I'll rebase shortly since you removed the dtrace file. As for include and some other directories, I did the "shotgun" approach to just make sure no files are needed. Since a few headers are ifdef'ed in the Makefile, autotools won't include them into a make dist (depending on what os you're on). I'll do a second run today and try to be more specific.